cmd/collector/app/server/grpc_test.go

```diff
@@ -71,21 +71,20 @@ func TestFailServe(t *testing.T) {
 func TestSpanCollector(t *testing.T) {
 	logger, _ := zap.NewDevelopment()
 	params := &GRPCServerParams{
-		Handler:       handler.NewGRPCHandler(logger, &mockSpanProcessor{}),
-		SamplingStore: &mockSamplingStore{},
-		Logger:        logger,
+		HostPort:                ":0",
+		Handler:                 handler.NewGRPCHandler(logger, &mockSpanProcessor{}),
+		SamplingStore:           &mockSamplingStore{},
+		Logger:                  logger,
+		MaxReceiveMessageLength: 1024 * 1024,
```

Comment:
made this the only place where param is initialized in tests, to have code coverage for the if statement